mars/services/scheduling/worker/execution.py

```diff
@@ -167,15 +167,10 @@ async def _prepare_input_data(self, subtask: Subtask, band_name: str):
         storage_api = await StorageAPI.create(
             subtask.session_id, address=self.address, band_name=band_name
         )
-        pure_dep_keys = set()
```

Comment:
Please explain the problem why this logic does not work?

Reply:
`pure_dep_keys` is only updated but not used.